tools/testing-utils/lazySingletonStyleTagModule.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
why not `.ts`?

Reply:
I couldn't import a `*.ts` file directly from a package, only `*.js` with sibling `*.d.ts` files.